hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/HsyncGenerator.java

```diff
@@ -53,32 +57,38 @@
     versionProvider = HddsVersionProvider.class,
     mixinStandardHelpOptions = true,
     showDefaultValues = true)
-public class HsyncGenerator extends HadoopNestedDirGenerator implements Callable<Void> {
+public class HsyncGenerator extends BaseFreonGenerator implements Callable<Void> {
   private static final Logger LOG = LoggerFactory.getLogger(HsyncGenerator.class);
 
   @CommandLine.ParentCommand
   private Freon freon;
 
+  @Option(names = {"--path"},
+      description = "Hadoop FS file system path. Use full path.",
+      defaultValue = "o3fs://bucket1.vol1")
+  private String rootPath;
+
   @Option(names = {"--bytes-per-write"},
       description = "Size of each write",
-      defaultValue = "1024")
+      defaultValue = "8")
   private int writeSize;
 
-  @Option(names = {"--number-of-files"},
-      description = "Number of files to run test.",
-      defaultValue = "1")
-  private int numberOfFiles;
```

Comment:
Test still uses this option:

https://github.com/apache/ozone/blob/3340947e8d2e5a96f88e8be320e64d4c46991674/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/freon/TestHsyncGenerator.java#L103

and fails:

```
AssertionFailedError: expected: <0> but was: <2>
  ...
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:531)
	at org.apache.hadoop.ozone.freon.TestHsyncGenerator.test(TestHsyncGenerator.java:106)
```